index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My CV Mao Suarez</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="contenedor">
+      <div class="titulo">
+        <img src="fotoCV.jpg" alt="Foto Perfil" />
+        <div class="titulo-text">
+          <h1>Mauricio Suarez Barrera</h1>
+          <h4>
+            Correo institucional: <spam>mauriciosuba@unisabana.edu.co</spam

```

Comment:
La etiqueta spam no existe, la etiqueta más parecida es span

```suggestion
            Correo institucional: <span>mauriciosuba@unisabana.edu.co</span

```